manywheel/build_rocm.sh

```diff
@@ -234,13 +234,19 @@ fi
 # No triton dependency for now on 3.12 since we don't have binaries for it
 # and torch.compile doesn't work.
 if [[ $(uname) == "Linux" && "$DESIRED_PYTHON" != "3.12" ]]; then
-    TRITON_SHORTHASH=$(cut -c1-10 $PYTORCH_ROOT/.ci/docker/ci_commit_pins/triton-rocm.txt)
+
```

Comment:
Related to https://github.com/pytorch/pytorch/pull/120744, why does this part not being moved to PyTorch?  It looks awkward to have triton Linux dependency setup on PyTorch why triton ROCm setup is is builder IMO

Reply:
Yes Agree. Let me move it there